### PR TITLE
Remove `_reset.scss`

### LINKED ISSUE #1453: Remove `_reset.scss`
We need to get rid of this file, for the migratio to tailwind. The styles defined in this Stylesheet affects all other components as we have `*`-Selectors in it.

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #1453

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1470"></a>

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
I think we should discuss if we want to keep all the resets (partially that arent resets anymore, but stylings) and remove / move them to other layers

File: src/lib/assets/global.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  a:focus {
+    @apply text-black underline decoration-transparent;
+  }
+
+  a[class] {
+    @apply no-underline;
+  }
+
+  ul:where([class]),
+  ol:where([class]) {
```
Comment (Cahllagerfeld) [inline src/lib/assets/global.css:43]: this

File: src/lib/assets/global.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+  body {
+    font-family: var(--font);
+    text-rendering: optimizeSpeed;
+    line-height: 1.5;
+    @apply m-0 min-h-screen text-dark-grey bg-sand-light text-p-medium;
+  }
+
+  a {
```
Comment (Cahllagerfeld) [inline src/lib/assets/global.css:30]: I feel like thats a default-styling. What do you think about this? When removing this, we probably need to take a look at EVERY other anchor-tag in the repo

**Review by Cahllagerfeld — COMMENTED** [review]
Two things I'm not sure about

File: src/lib/assets/global.css (hunk 2 above)
Comment (mikenikles) [reply src/lib/assets/global.css:30]: I agree, we'll have to do a thorough review. For now, I'd say lets merge it as is and look at it as part of the UI library creation.

File: src/lib/assets/global.css (hunk 1 above)
Comment (mikenikles) [reply src/lib/assets/global.css:43]: 👍 I can't think of a better solution until we have a `<List />` component in the UI library.

**Review by mikenikles — APPROVED** [review]
👍 2 `scss` files gone 💣

**Pull request merged**